src/Payload.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Minishlink\WebPush;
+
+use ErrorException;
+
+class Payload
+{
+    /**
+     * @var string
+     */
+    private $payload;
+    /**
+     * @var string
+     */
+    private $salt;
+    /**
+     * @var string
+     */
+    private $localPublicKey;
+
+    public function __construct(string $payload, string $salt, string $key)
+    {
+        $this->payload = $payload;
+        $this->salt = $salt;
+        $this->localPublicKey = $key;
+    }
+
+    /**
+     * @param Contracts\SubscriptionInterface $subscription
+     * @param string $payload
+     * @param int $padding
+     *
+     * @return Payload
+     * @throws ErrorException
+     */
+    public static function create(Contracts\SubscriptionInterface $subscription, string $payload, int $padding): Payload
```

Comment:
Why use this static method instead of a constructor

Reply:
Ah, that's right... I had other intentions for this class, but this can be changed now.

Reply:
The static factory method takes a different payload than the constructor - one that likely won't be able to be constructed by a consumer but rather only by the `PayloadBuilder`.